src/app/rpmostree-builtin-rebase.cxx

```diff
@@ -118,44 +118,44 @@ rpmostree_builtin_rebase (int             argc,
       if (argc == 3)
         revision = argv[2];
 
-      /* Canonicalize */
-      new_provided_refspec = new_refspec_owned =
-        rpmostree_refspec_canonicalize (new_provided_refspec, error);
-      if (!new_provided_refspec)
-        return FALSE;
+      new_provided_refspec = new_refspec_owned = g_strdup (new_provided_refspec);
     }
   else
     {
       if (opt_remote)
         {
           new_provided_refspec = new_refspec_owned =
-            g_strconcat (RPMOSTREE_REFSPEC_OSTREE_PREFIX, opt_remote, ":", opt_branch ?: "", NULL);
+            g_strconcat (opt_remote, ":", opt_branch ?: "", NULL);
```

Comment:
Worth trying to split out the "remove ostree://" bits as a separate PR?  I don't have a strong opinion but if it's not too hard I think it'd be helpful.

Reply:
Submitted https://github.com/coreos/rpm-ostree/pull/2980 :)